recipes/configs/generate.yaml

```diff
@@ -16,6 +16,7 @@ checkpointer:
 
 device: cuda
 dtype: bf16
+quantization_mode: f16a4w
```

Comment:
Curious why we disable quantization by default for eval but enable it by default for generation?

Reply:
oh sorry this should be null